build.gradle

```diff
@@ -1,6 +1,7 @@
 apply plugin: 'java'
 apply plugin: 'maven'
 apply plugin: 'signing'
+apply plugin: 'eclipse'
```

Comment:
I usually add it to remove dependency on the IDE - otherwise developers tend to commit IDE specific files since they find it hard to generate the workspace/project and being part of the build it is still safe to ignore. 

But if this is your policy I will take it out - I will commit a fix in a few minutes.